apps/ledger-live-desktop/src/renderer/hooks/useDeeplinking.ts

```diff
@@ -93,6 +94,18 @@ export function useDeepLinkHandler() {
       const query = Object.fromEntries(searchParams);
```

Comment:
I know this is unrelated so feel free to ignore for this PR. But searchParams looks to be an instance of [URLSearchParams](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams). Could we do something like the following below instead of using `Object.fromEntries`.


```
const ajsPropSource = searchParams.get("ajs_prop_source")
```

Reply:
Or maybe even use the `has` method so theres no need to store `ajs_prop_source` in  a const.

```
if (searchParams.has("ajs_prop_source")) {
```